Taarafo.Core/Brokers/Storages/StorageBroker.cs

```diff
@@ -33,13 +34,12 @@ private async ValueTask<T> InsertAsync<T>(T @object)
 		private async ValueTask<T> SelectAsync<T>(params object[] objectIds) where T : class =>
 			await FindAsync<T>(objectIds);
 		
-		protected override void OnModelCreating(ModelBuilder modelBuilder)
+		private IQueryable<T> SelectAll<T>() where T : class
```

Comment:
You also need to add `optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);` to `protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)` as we do not want to change tracking read operations.

change to 
```
protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {
            optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
            string connectionString = this.configuration.GetConnectionString(name: "DefaultConnection");
            optionsBuilder.UseSqlServer(connectionString);
        }
```